logstash-core/src/main/java/org/logstash/instrument/witness/QueueWitness.java

```diff
@@ -49,10 +95,9 @@ public void type(String type) {
 
         /**
          * Gets the type of queue
-         *
          * @return the queue type.
          */
-        public String type() {
+        public String type(){
```

Comment:
@jakelandis you made this formatting change (removing the space) in many spots, sure you want this? I think the majority of our codebase has that space there right now? :D